roles/common/tasks/service_tests.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Verify service is running
+  ansible.builtin.shell:
+    cmd: |
+      oc get service -n "{{ service_nspace }}" "{{ service_name }}"
```

Comment:
To align with the rest of the tests, this should loop through a list of test items, so that multiple services can be tested together